prow/images/OWNERS

```diff
@@ -0,0 +1,10 @@
+filters:
+  ".*":
+    labels:
+      - prow/images
+
+reviewers:
+  - tooling-reviewers
+
+approvers:
+  - tooling-approvers
```

Comment:
We should define separate OWNERS for some subdirectories here as well. Following are images for which we should not be approvers.
buildpack-java-node
buildpack-java
buildpack-node-chromium
buildpack-node

Reply:
Tooling-team should be the approver of all images. It is related to the tooling and works directly on Prow. For the reviewer part the console team should review these images, that's true, but I would leave this until the approval process will have a green light

Reply:
Don't agree with you. These are images for specific tests. Tooling team is not responsible for all tests and we don't have required knowledge from all technologies used in tests. At present and according to the proposal members of developers group have rights to approve changes here as well. This change should not increase our workload for the time of tests.